task-launcher/src/styles/abstracts/_variables.scss

```diff
@@ -14,3 +14,4 @@ $button-secondary-inner-shadow: 0px 0px 12px 0px;
 $response-size-default: clamp(120px, 12vw, 150px);
 $response-size-sm: clamp(100px, 20vw, 150px);
 $response-size-s: 25vw;
+$response-size-large: clamp(70px, 20vw, 250px)
```

Comment:
This looks still the same as `$response-size-sm`, only clamped at different pixels. We should probably increase the `20vw` values to justify the name also since it defined `image-medium`, it should probably be named `$response-size-m`.

Reply:
I think that value has to be at `20vw` or lower in order to prevent the buttons from going out of screen as far as I can tell, but I changed the name to `$response-size-m` (I agree that makes more sense)